debian/postinst

```diff
@@ -3,14 +3,16 @@
 set -e
 
 . /etc/os-release  # For VERSION_ID
+PREVIOUS_PKG_VER=$2
 
 ESM_KEY_ID_TRUSTY="56F7650A24C9E9ECF87C4D8D4067E40313CB4B13"
 ESM_APT_GPG_KEY="/etc/apt/trusted.gpg.d/ubuntu-esm-v2-keyring.gpg"
 ESM_APT_GPG_KEY_OLD="/etc/apt/trusted.gpg.d/ubuntu-esm-keyring.gpg"
-ESM_APT_SOURCE_FILE_OLD="/etc/apt/sources.list.d/ubuntu-esm-trusty.list"
-ESM_APT_SOURCE_FILE="/etc/apt/sources.list.d/ubuntu-esm-infra-trusty.list"
-ESM_APT_PREF_FILE_OLD="/etc/apt/preferences.d/ubuntu-esm-trusty"
-ESM_APT_PREF_FILE="/etc/apt/preferences.d/ubuntu-esm-infra-trusty"
+ESM_APT_SOURCE_FILE_PRECISE="/etc/apt/sources.list.d/ubuntu-esm-precise.list"
+ESM_APT_SOURCE_FILE="/etc/apt/sources.list.d/ubuntu-esm-trusty.list"
```

Comment:
Nit: Should we encode trusty in these variable names too?

Reply:
I didn't only because of line length and super long variable names both in declaration and in conditionals statements. At risk of the var name being near as long as the full path name, I'd like to avoid that in this case.

Reply:
Fixing this in the latest rev. As there are some other var name changes that would further muddy the waters.